tracer/src/Datadog.Tracer.Native/clr_helpers.cpp

```diff
@@ -1010,13 +1013,34 @@ HRESULT FunctionLocalSignature::TryParse(PCCOR_SIGNATURE pbBase, unsigned len, s
     return S_OK;
 }
 
+shared::WSTRING GetStringValueFromBlob(PCCOR_SIGNATURE& signature)
+{
+    // If it's null
+    if (*signature == UINT8_MAX)
+    {
+        signature += 1;
+        return shared::WSTRING();
+    }
+
+    // Read size and advance
+    ULONG size{CorSigUncompressData(signature)};
+    shared::WSTRING wstr;
+    wstr.reserve(size);
+
+    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+    std::wstring temp =
+        converter.from_bytes(reinterpret_cast<const char*>(signature), reinterpret_cast<const char*>(signature) + size);
+    wstr.assign(temp.begin(), temp.end());
+    signature += size;
+    return std::move(wstr);
```

Comment:
you can avoid the `std::move`, this break optimization 
https://stackoverflow.com/questions/17473753/c11-return-value-optimization-or-move